tuta-sdk/rust/src/entity_client.rs

```diff
@@ -43,18 +50,23 @@ impl EntityClient {
 
 impl EntityClient {
     /// Gets an entity/instance of type `type_ref` from the backend
-    pub async fn load_element(
+    pub async fn load_element<'a, T: Entity + Deserialize<'a>>(
```

Comment:
I think we should keep the untyped `EntityClient` as is and use it from both typed one (the one you implement) and one with encryption (one for later)

Reply:
Fixed.